nlmod/util.py

```diff
@@ -15,10 +15,50 @@
 logger = logging.getLogger(__name__)
 
 
+class LayerError(Exception):
+    pass
+
+
+class MissingValueError(Exception):
```

Comment:
Maybe add a description where we use these errors? Or move them to the appropriate file?

Reply:
I agree